src/rn-onesignal.d.ts

```diff
@@ -0,0 +1,268 @@
+// modified a little bit from https://github.com/OneSignal/react-native-onesignal/issues/453#issuecomment-404003702
```

Comment:
Would you put these codes in `environment.d.ts` where we put all `typescript` declare statement?

![image](https://user-images.githubusercontent.com/27461460/76811638-a17b7700-6835-11ea-9b69-eb569eecd08a.png)

As shown above.